nixos/modules/services/web-apps/pixelfed.nix

```diff
@@ -0,0 +1,366 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pixelfed;
+  opt = options.services.pixelfed;
+
+  user = "pixelfed";
```

Comment:
maybe we should expose the user through the module if we decide to go with the user approach